core/jazz_logs/index.js

```diff
@@ -106,14 +111,7 @@ module.exports.handler = (event, context, cb) => {
 
 			logger.info("QueryObj: " + JSON.stringify(querys));
 
-			var servCategory = [];
-
-			if (categoryType.toLowerCase() == 'api') {
-				servCategory = ["apilogs", "applicationlogs"];
-			} else if (categoryType.toLowerCase() == 'function') {
-				servCategory = ["applicationlogs"];
-			}
-
+			var servCategory = assetType ? [`${assetType}`] : [];
```

Comment:
Configure the mapping b/w asset_type & ES index. Use the index mapped with user provided asset_type.

Reply:
Updated

Reply:
changes look good.